vet-service/src/main/java/com/petclinic/vet/servicelayer/RatingServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.petclinic.vet.servicelayer;
+
+import com.petclinic.vet.dataaccesslayer.RatingRepository;
+import com.petclinic.vet.exceptions.ExistingVetNotFoundException;
+import com.petclinic.vet.util.EntityDtoUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@Service
+public class RatingServiceImpl implements RatingService {
+    @Autowired
```

Comment:
Use constructor injection instead of field injection as mentioned in previous comments.